src/common/socket.hpp

```diff
@@ -18,102 +18,101 @@
 #include "malloc.hpp"
-#include "timer.hpp" // t_tick
+#include "timer.hpp"  // t_tick
 
 #ifndef MAXCONN
 	#define MAXCONN FD_SETSIZE
 #endif
 
-#define FIFOSIZE_SERVERLINK 256*1024
+#define FIFOSIZE_SERVERLINK 256 * 1024
 
 // socket I/O macros
 #define RFIFOHEAD(fd)
-#define WFIFOHEAD( fd, size ) \
-	do{ \
-		if( ( fd ) && session[( fd )]->wdata_size + ( size ) > session[( fd )]->max_wdata ){ \
-			_realloc_writefifo( ( fd ), ( size ), ALC_MARK ); \
-		} \
-	}while( false )
-#define RFIFOP(fd,pos) (session[fd]->rdata + session[fd]->rdata_pos + (pos))
-#define WFIFOP(fd,pos) (session[fd]->wdata + session[fd]->wdata_size + (pos))
-
-#define RFIFOCP(fd,pos) ((char*)RFIFOP(fd,pos))
-#define WFIFOCP(fd,pos) ((char*)WFIFOP(fd,pos))
-#define RFIFOB(fd,pos) (*(uint8*)RFIFOP(fd,pos))
-#define WFIFOB(fd,pos) (*(uint8*)WFIFOP(fd,pos))
-#define RFIFOW(fd,pos) (*(uint16*)RFIFOP(fd,pos))
-#define WFIFOW(fd,pos) (*(uint16*)WFIFOP(fd,pos))
-#define RFIFOL(fd,pos) (*(uint32*)RFIFOP(fd,pos))
-#define WFIFOL(fd,pos) (*(uint32*)WFIFOP(fd,pos))
-#define RFIFOF(fd,pos) (*(float*)RFIFOP(fd,pos))
-#define WFIFOF(fd,pos) (*(float*)WFIFOP(fd,pos))
-#define RFIFOQ(fd,pos) (*(uint64*)RFIFOP(fd,pos))
-#define WFIFOQ(fd,pos) (*(uint64*)WFIFOP(fd,pos))
+#define WFIFOHEAD(fd, size)                                                          \
+	do {                                                                             \
+		if ((fd) && session[(fd)]->wdata_size + (size) > session[(fd)]->max_wdata) { \
+			_realloc_writefifo((fd), (size), ALC_MARK);                              \
+		}                                                                            \
+	} while (false)
+#define RFIFOP(fd, pos) (session[fd]->rdata + session[fd]->rdata_pos + (pos))
+#define WFIFOP(fd, pos) (session[fd]->wdata + session[fd]->wdata_size + (pos))
+
+#define RFIFOCP(fd, pos) ((char*)RFIFOP(fd, pos))
+#define WFIFOCP(fd, pos) ((char*)WFIFOP(fd, pos))
+#define RFIFOB(fd, pos) (*(uint8*)RFIFOP(fd, pos))
+#define WFIFOB(fd, pos) (*(uint8*)WFIFOP(fd, pos))
+#define RFIFOW(fd, pos) (*(uint16*)RFIFOP(fd, pos))
+#define WFIFOW(fd, pos) (*(uint16*)WFIFOP(fd, pos))
+#define RFIFOL(fd, pos) (*(uint32*)RFIFOP(fd, pos))
+#define WFIFOL(fd, pos) (*(uint32*)WFIFOP(fd, pos))
+#define RFIFOF(fd, pos) (*(float*)RFIFOP(fd, pos))
+#define WFIFOF(fd, pos) (*(float*)WFIFOP(fd, pos))
+#define RFIFOQ(fd, pos) (*(uint64*)RFIFOP(fd, pos))
+#define WFIFOQ(fd, pos) (*(uint64*)WFIFOP(fd, pos))
 #define RFIFOSPACE(fd) (session[fd]->max_rdata - session[fd]->rdata_size)
 #define WFIFOSPACE(fd) (session[fd]->max_wdata - session[fd]->wdata_size)
 
-#define RFIFOREST(fd)  (session[fd]->flag.eof ? 0 : session[fd]->rdata_size - session[fd]->rdata_pos)
-#define RFIFOFLUSH(fd) \
-	do { \
-		if(session[fd]->rdata_size == session[fd]->rdata_pos){ \
+#define RFIFOREST(fd) (session[fd]->flag.eof ? 0 : session[fd]->rdata_size - session[fd]->rdata_pos)
+#define RFIFOFLUSH(fd)                                            \
```

Comment:
This is [`AlignEscapedNewlines`](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#alignescapednewlines), honestly I like the right align, it makes it obvious how far the macro goes.

Reply:
To me it makes it harder to review here on github, especially because it uses a mix of tabs and spaces.
If you see a \ at the end of the line you know that it still continues. Which is fair and easy I guess.